autoload/lsp/lspserver.vim

```diff
@@ -1523,8 +1522,6 @@ export def NewLspServer(name_arg: string, path_arg: string, args: list<string>,
     processNotif: function(handlers.ProcessNotif, [lspserver]),
     processRequest: function(handlers.ProcessRequest, [lspserver]),
     processMessages: function(handlers.ProcessMessages, [lspserver]),
-    getDiagByPos: function(diag.GetDiagByPos, [lspserver]),
-    getDiagsByLine: function(diag.GetDiagsByLine, [lspserver]),
```

Comment:
These methods doesn't really make sense anymore as the diagnostics aren't really tied to a single LSP server.